Pages/eduandwork.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>Education and Work</title>
+  <link href= "styles.css" rel= "stylesheet"/>
+</head>
+
+<body>
+
+  <header>
+    <div class= "header-inline">
+      <img src= "https://www.deltavcodeschool.com/wp-content/uploads/deltav-logo.jpg"/>
+      <nav class= "primary-nav">
+        <ul>
+          <li><a href= "index.html">Home</a></li>
+          <li><a href= "eduandwork.html">Education and Work Experience</a></li>
+          <li><a href= "funfacts.html">Fun Facts</a></li>
+          <li><a href= "gamepage.html">Guessing Game</a></li>
+        </ul>
+      </nav>
+    </div>
+    <nav class= "social-nav">
+      <ul>
+        <li><a href= "https://www.facebook.com/dustin.yelland">Facebook</a></li>
+        <li><a href= "https://twitter.com/dustinoid2000">Twitter</a></li>
+        <li><a href= "https://github.com/dustinyschild">Github</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <h1>Education</h1>
+  <p>
+    There's not really much to say here,
+    I graduated from high school in May
+    of 2014 and was accepted to the Iowa
+    State University engineering program.
+    However, I did not go and decided to
+    go straight for a full time job.
+  </p>
+
+  <h1>Working Experience</h1>
```

Comment:
General rule is that a page should have a single `<h1>`, usually matching its `<title>`. This should probably be an `<h2>`.